packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -4,204 +4,549 @@
+              ],
+              onChanged: (Object newSelection) {
+                setState(() {
+                  if (axis == direction) {
+                    mainAxisAlignment = newSelection;
+                  } else {
+                    crossAxisAlignment = newSelection;
+                  }
+                });
+              },
+            ),
+          )
+        ],
       ),
     );
   }
 
   @override
   Widget build(BuildContext context) {
-    final flexType = properties.type.toString();
     final theme = Theme.of(context);
     return Container(
       padding: const EdgeInsets.only(top: 32.0),
       child: Column(
         mainAxisAlignment: MainAxisAlignment.start,
         children: [
           Container(
-            margin: const EdgeInsets.only(bottom: 36.0),
+            margin: const EdgeInsets.only(bottom: 24.0),
             child: Text(
               'Story of the flex layout of your $flexType widget',
               style: theme.textTheme.headline,
               textAlign: TextAlign.center,
             ),
           ),
-          Expanded(
+          Align(
+            alignment: Alignment.centerRight,
+            child: _buildAxisAlignmentDropdown(Axis.horizontal),
+          ),
+          Flexible(
             child: LayoutBuilder(builder: (context, constraints) {
-              final children = _visualizeChildren(context);
-              final flexDirectionWrapper = Flex(
-                direction: properties.direction,
-                children: children,
-              );
-              final childrenVisualizerWidget = Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  Container(
-                    child: Text(
-                      properties.type.toString(),
+              final maxHeight = constraints.maxHeight * 0.95;
+              final maxWidth = constraints.maxWidth * 0.95;
+              const topArrowIndicatorHeight = 32.0;
+              const leftArrowIndicatorWidth = 32.0;
+              const margin = 8.0;
+              return Container(
+                constraints:
+                    BoxConstraints(maxWidth: maxWidth, maxHeight: maxHeight),
+                child: Stack(
+                  children: <Widget>[
+                    Align(
+                      alignment: Alignment.centerLeft,
+                      child: Container(
+                        margin: const EdgeInsets.only(
+                          top: topArrowIndicatorHeight,
+                          left: leftArrowIndicatorWidth + margin,
+                        ),
+                        child: WidgetVisualizer(
+                          title: flexType,
+                          hint: Container(
+                            padding: const EdgeInsets.all(4.0),
+                            child: Text(
+                              'Total Flex Factor: ${properties.totalFlex}',
+                              textScaleFactor: 1.2,
```

Comment:
looks like you have a text style with scale factor 1.2 you are using a lot of places. Either define a constant or better yet use a text style from the theme that is semantically consistent with what you need here.